applications/GeoMechanicsApplication/custom_conditions/U_Pw_condition.hpp

```diff
@@ -106,8 +108,9 @@ class KRATOS_API(GEO_MECHANICS_APPLICATION) UPwCondition : public Condition
     {
         KRATOS_SERIALIZE_LOAD_BASE_CLASS(rSerializer, Condition)
     }
+};
 
-}; // class UPwCondition.
+// class UPwCondition.
```

Comment:
Same here with the strange layout (and multiple other classes, I won't mention it the rest of the times)

Reply:
See my previous reply.